mammoth/utils/parse.py

```diff
@@ -27,7 +36,7 @@ def _validate_adapters(cls, opts):
         """Parse corpora specified in data field of YAML file."""
         if not opts.adapters:
             return
-        adapter_opts = yaml.safe_load(opts.adapters)
+        adapter_opts = yaml_or_dict(opts.adapters, name='opts.adapters')
```

Comment:
not sure about this `name` thing — I get that you want to display a less cryptic message, but in practice we devs are the only ones who'll ever get to see that `TypeError` ... and you're using that function once? the stacktrace should be unambiguous enough.

Reply:
I forgot to apply this to the other similar locations where `yaml.safe_load` is applied. Fixed now.

This crashed when loading opts.adapters from a checkpoint. I'm not sure why those other usages didn't crash earlier, but better safe than sorry.